Added support for opening folder in same or new window.

mukundshah opened the pull request.

mukundshah committed:
Support for opening folder in same or new window

mukundshah committed:
2.3.0

sirfuzzalot commented:
Hello @mukundshah . I'm trying to figure out if you changed anything besides updating the version number and reformatting three files to a different style. Let me know if you did make other changes. If so pull those changes out and remove the reformatting please. If I do want to reformat everything I will do a reformat pass on all the code and the PR will be labelled as such. Thanks.

mukundshah commented:
Hi @sirfuzzalot .
I added an option in the right-click menu to open the folder in same or new window.
![image](https://user-images.githubusercontent.com/39938037/146665183-8c41e916-ad15-4880-a8bd-dfd2d0d0dd16.png)

I use different formatting style. Therefore, the formatting of the files I have touched have been altered. Those formatting were involuntary. And because of that, the diff is a mess. I wanted to reverse them, but without a prettier config, it would have taken long time. I have added an issue to add a prettier config file to the repo.

mukundshah commented:
Hi @sirfuzzalot. I opened another PR with correct required changes only. #19
